doc/src/manual/interfaces.md

```diff
@@ -231,8 +231,9 @@ arrays are simple: just define `getindex(A::ArrayType, i::Int)`.  When the array
 indexed with a multidimensional set of indices, the fallback `getindex(A::AbstractArray, I...)()`
 efficiently converts the indices into one linear index and then calls the above method. `IndexCartesian()`
 arrays, on the other hand, require methods to be defined for each supported dimensionality with
-`ndims(A)``Int` indices.  For example, the builtin `SparseMatrixCSC` type only supports two dimensions,
-so it just defines `getindex(A::SparseMatrixCSC, i::Int, j::Int)()`.  The same holds for `setindex!()`.
+`ndims(A)``Int` indices.  For example, the builtin [`SparseMatrixCSC`](@ref) type only
```

Comment:
"built-in" seems to be most used in base. Note that this isn't a change I made; but I can update it.